zuul-core/src/main/java/com/netflix/zuul/netty/connectionpool/ClientTimeoutHandler.java

```diff
@@ -55,10 +56,10 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
         @Override
         public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise) throws Exception {
             try {
-                final Integer timeout = ctx.channel().attr(ORIGIN_RESPONSE_READ_TIMEOUT).get();
+                final Duration timeout = ctx.channel().attr(ORIGIN_RESPONSE_READ_TIMEOUT).get();
                 if (timeout != null && msg instanceof LastHttpContent) {
                     promise.addListener(e -> {
-                        LOG.debug("[{}] Adding read timeout handler: {}", ctx.channel().id(), timeout);
+                        LOG.debug("[{}] Adding read timeout handler: {}", ctx.channel().id(), timeout.toMillis());
```

Comment:
personal preference, but I find the non-`toMillis` more helpful to read.

Reply:
Was thinking it would be simpler since most configs are in millis.